script/cleanup.sh

```diff
@@ -19,6 +19,19 @@ if [ -d "/var/lib/dhcp" ]; then
 fi
 
 UBUNTU_VERSION=$(lsb_release -sr)
+if [[ ${UBUNTU_VERSION} == 16.04 ]] || [[ ${UBUNTU_VERSION} == 18.04 ]]; then
```

Comment:
This snippet was in there once, but has been removed in https://github.com/boxcutter/ubuntu/commit/300ad4ba01debf5e1db43e3e4d106fcf22b7bfcd (instead the pci slot number has been configured to ens32 in the vmx settings).

This won't help for the vmware desktop variants though, as they use "ens160" even though the explicit configuration in the vmx settings. The reason / root cause for this is that the VMWare Desktop VMs are configured with a different (i.e. non default) network adapter type "vmxnet3" here:
https://github.com/boxcutter/ubuntu/blob/master/tpl/vagrantfile-ubuntu1804-desktop.tpl#L27

Removing this configuration makes the desktop variants use the default network adapter type which then correctly appears "ens32" in the VM. (this is how I would suggest to fix this :))

Reply:
Not sure if I would go for Option A) or rather B). Both are fixing the issue.

Considerations:

* vmxnet3 (as "ens160") is paravirtualized and said to be the most performant
* e1000 (as "ens32") is the one that is currently used for all the ubuntu server boxes

See also: https://kb.vmware.com/s/article/1001805

I have a slight tendency towards option B), as it would make the configuration of desktop and server VMs more uniform.

@rasa @misheska @fasmat thoughts / opinions / preferences?

Reply:
For now I implemented Option A) since this would at the moment also configure the network of desktop and server boxes in the same way.

Although I have a licensed vmware installation, I don't have a licensed vmware plugin for vagrant. So I currently cannot test Option B).

Reply:
FYI - I tested both, and both options would fix the issue (Option A by using "ens32" as we do for the server boxes, option B by adapting the network config files to configure "ens160" instead of "ens32")